package.json

```diff
@@ -13,9 +13,9 @@
     "build": "nebula build --core core",
     "sense": "nebula sense --meta resources/meta.json && shx cp resources/network_chart_v1.png sn-network-chart-ext",
     "eslint": "eslint src",
+    "test:unit": "jest",
```

Comment:
For some reason unit tests are not included in the pipeline. Could you add them as well while you are working on this?

Reply:
Good point. Let me add it.

Reply:
Done